src/components/tables/Details/ShardInformation.tsx

```diff
@@ -1,127 +1,194 @@
-                                <AccordionSummary
-                                    expandIcon={<ExpandMoreIcon />}
-                                >
-                                    <Typography>
-                                        <FormattedMessage id="detailsPanel.shardDetails.errorTitle" />
-                                    </Typography>
-                                </AccordionSummary>
-
-                                <AccordionDetails>
-                                    <Box sx={{ height: 250 }}>
-                                        <Editor
-                                            defaultLanguage=""
-                                            theme={
-                                                theme.palette.mode === 'light'
-                                                    ? 'vs'
-                                                    : 'vs-dark'
-                                            }
-                                            options={{
-                                                lineNumbers: 'off',
-                                                readOnly: true,
-                                                scrollBeyondLastLine: false,
-                                                minimap: {
-                                                    enabled: false,
-                                                },
+    }, [setTaskShards, getTaskShards, entityType, specs, shards]);
+
+    const changePage = (
+        _event: MouseEvent<HTMLButtonElement> | null,
+        newPage: number
+    ) => setPage(newPage);
+
+    return taskShards.length > 0 ? (
+        <>
+            <ShardErrors shards={taskShards} />
+
+            <Grid item xs={12}>
+                <TableContainer>
+                    <Table>
+                        <TableHead>
+                            <TableRow
+                                sx={{
+                                    background: theme.palette.background.paper,
+                                }}
+                            >
+                                <TableCell colSpan={columns.length}>
+                                    <Box
+                                        sx={{
+                                            width: '100%',
+                                            display: 'flex',
+                                            justifyContent: 'space-between',
+                                        }}
+                                    >
+                                        <Typography
+                                            component="span"
+                                            align="center"
+                                            sx={{ width: '100%' }}
+                                        >
+                                            <FormattedMessage id="detailsPanel.shardDetails.title" />
+                                        </Typography>
+
+                                        <Box
```

Comment:
As requested, a link to the shard documentation is provided in the far right end of the shard information table header. The sticky link maintained its positioning when the screen was resized.